charts/prometheus/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v2
 name: prometheus
 appVersion: 2.26.0
-version: 14.8.1
+version: 14.8.2
```

Comment:
New feature should raise minor version

Reply:
That's partially true. You can treat it also as bug-fix, because dictionary was mentioned in values.yaml and not used, see:
https://github.com/prometheus-community/helm-charts/blob/main/charts/prometheus/values.yaml#L412
Anyway I will not insist to not change it ;).

Reply:
@monotek can you take a look?